src/applyLockMass.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+const analyseMF = require('chemcalc').analyseMF;
+
+/**
+ * Recalculates series for GC/MS with lock mass
+ * @param {Chromatogram} chromatogram - GC/MS chromatogram where make the peak picking
+ * @param {string|Array<string>} molecularForm - Reference molecular formula(s)
```

Comment:
we are used to speak about 'mf' and not 'molecularForm'. You can just use mf for the name of the variable